src/router/index.js

```diff
@@ -1252,16 +1270,55 @@ const router = new Router({
       component: loggingRoom,
       meta: {
         title: 'Logging Rooms',
-        auth: true
+        auth: true,
+        breadcrumb: [
+          { name: 'Home', link: 'feeds' },
+          { name: 'logging Room', link: 'loggingRoom' }
```

Comment:
`logging -> Logging` in `name` field

Please also check the rest of this file for this occurrence

Reply:
Resolved, please check again.